src/components/ProfitAndLossSummaries/ProfitAndLossSummaries.tsx

```diff
@@ -54,149 +56,113 @@ const buildMiniChartData = (scope: Scope, data?: ProfitAndLoss) => {
   return items.slice()
 }
 
+const SECTION_CLASS_NAME = 'Layer__ProfitAndLossSummaries'
+const SECTION_CLASS_NAMES = `${SECTION_CLASS_NAME} Layer__component`
+
 type ProfitAndLossSummariesProps = {
-  vertical?: boolean
   actionable?: boolean
+  stringOverrides?: ProfitAndLossSummariesStringOverrides
+  chartColorsList?: string[]
+  slots?: {
+    unstable_AdditionalListItems?: [ReactNode]
```

Comment:
The goal here, as I understand it, is to allow us internally to set `additionalListItem` (for use cases like the `<TransactionsToReviewCard />`), but we don't want customers to use this slot. Which makes sense!

I think we could accomplish this by having a very simple wrapper component which _doesn't_ include this optional prop but otherwise delegates entirely to this internal version. Then we only publish/expose the wrapper so customers only see a version where `unstableAdditionalListItems` isn't even a prop.